tmp_output/AirQuality/Current.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
You shouldn't commit this file to the repo.

Reply:
Right. Forgot to delete them